recipe/meta.yaml

```
@@ -145,11 +146,12 @@ outputs:
         - json-c   # [not win]
         - libiconv
         - libkml
-        - libpq
+        - {{ pin_compatible('libpq', max_pin='x.x.x') }}
```

```suggestion
        - libpq
```
libpq has a run_exports: https://github.com/AnacondaRecipes/postgresql-feedstock/blob/PKG-3916/recipe/meta.yaml#L102
Pinning too strictly will also trigger unnecessary rebuilds later on.

---

Is the right way to handle that to include it without pinning and let the `run_exports` set the versions or to just omit it from the runtime deps in this recipe all together? I'm seeing it both ways in other recipes.